src/tiledb/cloud/bioimg/helpers.py

```diff
@@ -34,3 +35,35 @@ def serialize_filter(filter):
         return filter_dict
     else:
         raise TypeError
+
+
+def is_folder(path: str) -> bool:
+    return path.endswith("/")
+
+
+def validate_io_paths(source: Sequence[str], output: Sequence[str]):
+    if not isinstance(source, list) or not isinstance(output, list):
```

Comment:
instead of `not isinstance(source, list)`, it's probably better to check for `isinstance(source, str)`, so that you're not *exclusively* allowing lists (e.g., you could pass a tuple).

Reply:
Will totally remove after all before the validation we cast to list if the input is a str so the check is not useful.